index.html

```diff
@@ -50,31 +50,33 @@
   <!-- End Matomo Code -->
 </head>
 <body>
-	<nav>
-		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
-      <a class="language-menu--item" href="/" itemprop='url'>
-        <span itemprop="name">English</span>
-      </a>
-      <a class="language-menu--item" href="/es/" itemprop='url'>
-        <span itemprop="name">Español</span>
-      </a>
-		</div>
+  <header>
+  	<nav>
       <a class="logo-btn" href="/" itemprop="url">
         <img
-				  srcset="/app/assets/images/logo-macile-167x180.jpg 167w,
-				          /app/assets/images/logo-macile-116x125.jpg 116w,
-				          /app/assets/images/logo-macile-84x90.jpg 84w"
-				  sizes="(min-width: 800px) 167px, (min-width: 500px) 116px, (min-width: 150px) 84px"
-				  alt="logo-macile">
-			</a>
-  </nav>
-	<!-- A <section> acts just like a <div> except it tells browsers and search engines that it is a section on our page -->
-	<section class="home-hero">
-		<div class="home-hero--inner">
-			<h1 class="home-hero--title">Our awesome graduation</h1>
-			<h2 class="home-hero--subtitle">Conquered steps are milestones on the journeys we're pursuing</h2>
-		</div>
-	</section>
+  				srcset="/app/assets/images/logo-macile-167x180.jpg 167w,
+  				        /app/assets/images/logo-macile-116x125.jpg 116w,
+  				        /app/assets/images/logo-macile-84x90.jpg 84w"
+  				sizes="(min-width: 800px) 167px, (min-width: 500px) 116px, (min-width: 150px) 84px"
+  				alt="logo-macile">
+  		</a>
+  		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+        <a class="language-menu--item" href="/" itemprop='url'>
+          <span itemprop="name">English</span>
+        </a>
+        <a class="language-menu--item" href="/es/" itemprop='url'>
+          <span itemprop="name">Español</span>
+        </a>
+  		</div>
+    </nav>
+  	<!-- A <section> acts just like a <div> except it tells browsers and search engines that it is a section on our page -->
```

Comment:
I think we can remove this comment on L72

Reply:
Can I remove all the comments that are based on lessons?

Reply:
@Morban15 Might as well!